backend/python/trading_env/lib/python3.11/site-packages/click/_termui_impl.py

```
@@ -0,0 +1,788 @@
+            url = unquote(url[7:])
+
+        return url
+
+    if sys.platform == "darwin":
+        args = ["open"]
+        if wait:
+            args.append("-W")
+        if locate:
+            args.append("-R")
+        args.append(_unquote_file(url))
+        null = open("/dev/null", "w")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)